### DISCUSSION TITLE
"Funding" count broken

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#4786] [General]
I write this here instead of opening an issue as I don't consider this a bug with MkDocs Material itself.

The number usually displayed right next to the "Funding" text, displaying the current amount of sponsor money received seems to be broken. There is no number displayed whatsoever.

![grafik](https://user-images.githubusercontent.com/11576465/208930126-703c1fe6-1d6f-45bc-b840-69649761731d.png)

However, the sidebar (ToC) is actually displaying the number, so this is only partially broken?

**Comment by squidfunk** [comment]
Thanks for reporting. This is due to the fact that the sidebar can now host HTML, which appears before the content area in the returned HTML, thus it's mounted first. Fixed in [`20683b5`](https://github.com/squidfunk/mkdocs-material-insiders/commit/20683b5fbc0b06ba58d78ec11a3bfa443e7a8a9c) – now content area and sidebar both show the funding 😊
